### PR TITLE
fix(storefront): BCTHEME-477 Add to cart button and Wishlist should be on one line on desktop

### PR DESCRIPTION
**Pull request by bc-yevhenii-buliuk**
@bc-alexsaiannyi @BC-tymurbiedukhin @yurytut1993 @bc-azvierieva 
#### What?

Fixed alignment of "Add to cart" and "Wishlist" buttons on all screen sizes.

#### Tickets / Documentation

- [BCTHEME-477](https://jira.bigcommerce.com/browse/BCTHEME-477)

#### Screenshots (if appropriate)

Снимок экрана 2021-05-05 в 16 04 20
Снимок экрана 2021-05-05 в 16 03 38
Снимок экрана 2021-05-05 в 15 58 47
Снимок экрана 2021-05-05 в 16 00 11

### CODE REVIEW

**Review by BC-tymurbiedukhin — APPROVED** [review]
LGTM

**Review by bc-alexsaiannyi — APPROVED** [review]

File: assets/scss/components/stencil/productView/_productView.scss [hunk 1]
```diff
@@ -290,7 +290,11 @@
     }
 
     .form {
-        display: inline;
+        display: inline-block;
+
+        @media (min-width: $screen-xxsmall) and (max-width: $screen-medium) {
+            display: inline;
+        }
```
Comment (yurytut1993) [inline assets/scss/components/stencil/productView/_productView.scss:297]: Can you please check your solution for products with a lot of variant option values? 
For example 
many-option-values

**Comment by yurytut1993** [comment]
@bc-yevhenii-buliuk also I think we have really small margin between add to cart and wishlist in some cases
small-margin

File: templates/components/products/add-to-cart.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div id="add-to-cart-wrapper" {{#unless product.can_purchase}}style="display: none"{{/unless}}>
+<div id="add-to-cart-wrapper" class='add-to-cart-wrapper' {{#unless product.can_purchase}}style="display: none"{{/unless}}>
```
Comment (BC-tymurbiedukhin) [inline templates/components/products/add-to-cart.html:1]: Please use double quotes

File: assets/scss/components/stencil/productView/_productView.scss [hunk 3]
```diff
@@ -347,10 +351,14 @@
...
 
+            @include breakpoint("medium") {
+                width: 60%;
+            }
+
             @include breakpoint("large") {
                 margin-right: spacing("half");
                 width: auto;
-            }
+            }
```
Comment (yurytut1993) [inline assets/scss/components/stencil/productView/_productView.scss:361]: Сan you please also remove this extra spaces after curly brace?

**Review by yurytut1993 — APPROVED** [review]

**Review by bc-azvierieva — APPROVED** [review]

**Pull request merged**